nemo/collections/vlm/llava_next/data/utils.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import torch
+import torch.nn.functional as F
+from megatron.core.packed_seq_params import PackedSeqParams
+
+
+def convert_to_packed_llava_next(
```

Comment:
is it possible to re-use the existing one? what's the major problem of re-using?

Reply:
It's mainly the predict seq length call difference and `final_padding_to` argument. I will work on merging this in the refactoring soon.